newt/vendor/mynewt.apache.org/newt/util/util.go

```diff
@@ -310,7 +310,12 @@ func ShellCommandLimitDbgOutput(
 	}
 
 	if err != nil {
-		return o, NewNewtError(string(o))
```

Comment:
Go isn't really my thing; should I remove the `/vendor` changes and let some re-vendoring process recreate those, or is it ok to just commit both?